warmrestart/warm_restart.cpp

```diff
@@ -96,6 +97,34 @@ bool WarmStart::checkWarmStart(const std::string &app_name, const std::string &d
     return true;
 }
 
+/*
+ * Get warmStartTimer for an application in a docker (default to be swss)
+ * return value 0 = OK,  -1 = failure
+ * timer value is saved to timer_value if succeed. timer value 0 is invalid.
+ */
+int WarmStart::getWarmStartTimer(uint32_t &timer_value, const std::string &app_name,
+    const std::string &docker_name)
+{
+    auto& warmStart = getInstance();
+    std::string timer_name = app_name + "_timer";
```

Comment:
So it is assumed that there is only one timer for each application like neighsyncd, bgp and etc..  Is there exception?